common/components/ui/Modal.scss

```diff
@@ -1,132 +1,130 @@
-@import "common/sass/variables";
-@import "common/sass/mixins";
+@import 'common/sass/variables';
+@import 'common/sass/mixins';
 
 $m-background: #fff;
 $m-window-padding: 20px;
 $m-header-padding: 15px;
 $m-header-height: 62px;
 $m-content-padding: 20px;
-$m-footer-height: 58px;
 $m-close-size: 26px;
 $m-anim-speed: 400ms;
 
 @keyframes modalshade-open {
-  0% {
-    opacity: 0;
-  }
-  70%,
-  100% {
-    opacity: 1;
-  }
+	0% {
+		opacity: 0;
+	}
+	70%,
+	100% {
+		opacity: 1;
+	}
 }
 
 @keyframes modal-open {
-  0%,
-  30% {
-    opacity: 0;
-    transform: translateX(-50%) scale(0.88);
-  }
-  100% {
-    opacity: 1;
-    transform: translateX(-50%) scale(1);
-  }
+	0%,
+	30% {
+		opacity: 0;
+		transform: translateX(-50%) scale(0.88);
+	}
+	100% {
+		opacity: 1;
+		transform: translateX(-50%) scale(1);
+	}
 }
 
 .Modalshade {
-  position: fixed;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  background-color: rgba(#000, 0.82);
-  z-index: $zindex-modal-background;
-  display: none;
-  animation: modalshade-open $m-anim-speed ease 1;
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	background-color: rgba(#000, 0.82);
+	z-index: $zindex-modal-background;
+	display: none;
+	animation: modalshade-open $m-anim-speed ease 1;
 
-  &.is-open {
-    display: block;
-  }
+	&.is-open {
+		display: block;
+	}
 }
 
 .Modal {
-  position: fixed;
-  top: 30px;
-  left: 50%;
-  max-width: 95%;
-  max-width: calc(100% - #{$m-window-padding * 2});
-  max-height: 95%;
-  max-height: calc(100% - #{$m-window-padding * 2});
-  background: $m-background;
-  border-radius: 4px;
-  transform: translateX(-50%);
-  z-index: $zindex-modal;
-  overflow: hidden;
-  display: none;
-  flex-direction: column;
-  animation: modal-open $m-anim-speed ease 1;
-  text-align: left;
+	position: fixed;
+	top: 30px;
+	left: 50%;
+	width: 95%;
+	max-width: 850px;
```

Comment:
If you want to target `max-width: 850px`, that should be done for the content of your particular modal. The idea here is that a modal is an unopinionated container, and should be able to be wider than that if someone needs it to be.

Reply:
I'll fix it up so its particular to the disclaimer modal.